test/KrylovKit.jl

```diff
@@ -34,6 +34,6 @@ function test_krylov_eigsolve(
     end
 end
 
-for dvec_type in (DVec, DVec2), address_type in (BSAdd64, BSAdd128, BoseFS{6,6,BSAdd64})
+for dvec_type in (DVec, DVec2), address_type in (BoseFS{6,6},)
```

Comment:
I am not familiar with the syntax `(BoseFS{6,6},)` , is this correct?

Reply:
`(a,)` is a single-element tuple. After this PR, you no longer have to specify the third parameter for `BoseFS` because it can be uniquely determined from the first two.

The `for`-loop is probably not necessary here though.

Reply:
Done, simplified the KrylovKit test.